src/components/RuntimeManager.js

```diff
@@ -292,7 +293,7 @@ export default class RuntimeManager extends PureComponent {
     const app = currentApp(apps)
 
     return h(Fragment, [
-      app && div({ style: { ...styles.container, borderRadius: 5, marginRight: '1.5rem' } }, [
+      app && isAnvil() && div({ style: { ...styles.container, borderRadius: 5, marginRight: '1.5rem' } }, [
```

Comment:
I think we should omit the `isAnvil` condition here, to remind people that they have an ongoing cost even if they go to another site skin.

Reply:
Are other resources shared between site skins? Would we still want the users to be able to launch Galaxy from a non-AnVIL skin?

Reply:
I think the idea was to only show the launch button on the Anvil skin to limit the visibility of the feature, but once an app is running, we should show it regardless to make sure people know what they're spending money on.